pandas/tests/extension/test_numpy.py

```diff
@@ -248,6 +248,10 @@ def test_repeat(self, data, repeats, as_series, use_numpy):
         # Fails creating expected
         super().test_repeat(data, repeats, as_series, use_numpy)
 
+    @pytest.mark.skip(reason="algorithms.diff skips PandasArray")
+    def test_diff(self, data, periods):
```

Comment:
why is this skipped?

Reply:
Either a bug or not implemented behavior in PandasArray. `PandasArray.shift()` doesn't allow changing the dtype from int to float, so `diff` (which uses shift) would fail.

Reply:
maybe xfail it then?